pkg/controller/plan/kubevirt.go

```diff
@@ -1159,8 +1159,12 @@ func (r *KubeVirt) dataVolumes(vm *plan.VMStatus, secret *core.Secret, configMap
 		annotations[AnnDefaultNetwork] = path.Join(
 			r.Plan.Spec.TransferNetwork.Namespace, r.Plan.Spec.TransferNetwork.Name)
 	}
-	// Set annotation for WFFC storage classes
-	annotations[planbase.AnnBindImmediate] = "true"
+	if r.Plan.Spec.Warm || !r.Destination.Provider.IsHost() || r.Plan.IsSourceProviderOCP() {
```

Comment:
maybe a comment we should drop this condition once 4.14 EOL?

Reply:
@liranr23 even then it would be better not to request CDI to immediately bind but to wait for the conversion pod
in the long run, we should see how it would work with volume populators that run by CDI and virt-v2v on el9 should be started by CDI in a similar way in cold migrations to a remote cluster